tests/netebpfext_unit/netebpfext_unit.cpp

```diff
@@ -60,8 +60,8 @@ netebpfext_unit_invoke_program(
 // Netebpfext is ready for us to attach to it as if we were ebpfcore.
 NTSTATUS
 netebpf_unit_attach_extension(
-    _In_ HANDLE nmr_binding_handle,
-    _In_ void* client_context,
+    _In_ const HANDLE nmr_binding_handle,
```

Comment:
```suggestion
    _In_ HANDLE nmr_binding_handle,
```
revert this line since const here just makes the input argument value (not any memory pointed to by the handle) immutable which is not necessary